docs/sources/whatsnew/whats-new-in-v9-5.md

```diff
@@ -0,0 +1,180 @@
+
+### Access a dashboard or panel from an alert rule
+
+Navigate from an alert rule straight to a dashboard or a panel associated with the alert rule to visualize your alerting data.
+
+{{< figure src="/media/docs/alerting/dashboard-alert-rule.png" max-width="750px" caption="Access a dashboard or panel from an alert rule" >}}
+
+### Preview queries for recording rules
+
+Visualize queries when creating or editing recording rules, so you can see the results of your query before saving your recording rule.
+
+{{< figure src="/media/docs/alerting/preview-queries-recording-rule.png" max-width="750px" caption="Preview queries for recording rules" >}}
+
+### Wait before firing alerts when no data is returned
+
+Alert rules that are configured to fire when an evaluation returns no data now only fire when the entire duration of the evaluation period has finished. This means that rather than immediately firing when the alert rule condition is breached, the alert rule waits until the time set in the **For** field has finished and then fires, reducing alert noise and allowing for temporary data availability issues.
+
+### Improved Notification policies view
+
+Updates to the Notification policies view make it easier to use and manage in the following ways:
+
+- View default policy and nested policies at a glance
+- New tab for mute timings
+- View alert instances for each policy
+- View contact points and which integrations are configured for each policy
+- View inherited properties on nested policies
+- Search for label matchers and for contact points to see which notifications are going where
+
+{{< figure src="/media/docs/alerting/notification-policies-view.png" max-width="750px" caption="Improved Notification policies view" >}}
+
+### Guidance for configuring your Alertmanager
+
+Get additional help while configuring your Alertmanager. If you enter an invalid Alertmanager configuration, an error message displays, and you can choose from a previous working configuration to restart it.
+
+## InfluxDB plugin database field deprecation
+
+The `database` field in the provisioning file has been deprecated.
+This information will be stored in `jsonData` with `dbName` field.
+The `database` field will be removed in the future to make InfluxDB consistent with other data sources.
+For more information and examples please refer to the [InfluxDB Provisioning docs]({{< relref "../datasources/influxdb/#provision-the-data-source" >}}).
+
+## Auth: Lock organization roles synced from auth providers
+
+_Generally available in all editions of Grafana._
+
+Grafana v9.4 provided the ability to configure synchronization of organization roles for each OAuth provider. With synchronization on, the organization role was applied to the user from the OAuth provider upon signing in. After signing in, during the session, you could change the user’s organization role.
```

Comment:
@danielkenlee - Does this edit work? Want to make this clearer.

```suggestion
Grafana v9.4 provided the ability to configure synchronization of organization roles for each OAuth provider. With synchronization on, the organization role was applied to the user from the OAuth provider upon signing in. However, after the user signed in, you could still change the user’s organization role during the session.
```

Reply:
Much clearer, thanks Isabel!